roles/ocm-labels/tests/inventory

```diff
@@ -0,0 +1,10 @@
+[hub_cluster]
+test-cluster-1 kubeconfig=/path/to/kubeconfig1
+
+[managed_clusters]
+test-cluster-2 kubeconfig=/path/to/kubeconfig2
+test-cluster-3 kubeconfig=/path/to/kubeconfig3
+test-cluster-4 kubeconfig=/path/to/kubeconfig4
```

Comment:
in this usecase the task is only interacting with the hub is the kubeconfig for the managedcluster required?

Reply:
You are correct, these entries are not necessary because labelling only interacts with the hub_cluster.

Reply:
I'm going to leave this example as is even though it is not necessary for the role. I feel it gives a full picture of the hub/spoke deployment.